src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -147,6 +172,17 @@ public OpenSearchClient initializeJavaClient(ExtensionSettings settings) {
         return initializeJavaClient(settings.getOpensearchAddress(), Integer.parseInt(settings.getOpensearchPort()));
     }
 
+    /**
+     * Initializes an OpenAsyncSearchClient using OpenSearch JavaClient
+     *
+     * @param settings The Extension settings
+     * @return The SDKClient implementation of OpenSearchAsyncClient. The user is responsible for calling
+     *         {@link #doCloseJavaClient()} when finished with the client
```

Comment:
The user may be responsible for closing this client, but this points them to the wrong client to close. :-)

Reply:
If we're using the same close method for both clients that should be documented here as well.